src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -138,6 +138,13 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// </summary>
         public bool StartFromBeginning { get; set; } = false;
 
+        /// <summary>
+        /// Optional.
+        /// Gets or sets the DateTime from which feed should start from. 
+        /// This is useful when you need to skip past and start from a checkpoint. It is best to use UTC date / time format to avoid any discrepencies.
```

Comment:
I would not mention anything about checkpoints and the UTC timezone should be a `<remark>`.